### PR TITLE
DIRECTOR & VIDEO: Fully working undome!

### PR DESCRIPTION
**Pull request by hari01584**
UnDo me of meet-mediaband (win), is working with all playback controls, rate, etc! Additionally some bugs are also fixed.

### CODE REVIEW

File: video/video_decoder.cpp [hunk 1]
```diff
@@ -631,10 +632,11 @@ Audio::Timestamp VideoDecoder::FixedRateVideoTrack::getDuration() const {
 }
 
 VideoDecoder::AudioTrack::AudioTrack(Audio::Mixer::SoundType soundType) :
-		_volume(Audio::Mixer::kMaxChannelVolume),
```
Comment (sev-) [inline video/video_decoder.cpp:634]: Please revert this unnecessary indentation change

**Review by sev- — CHANGES_REQUESTED** [review]
Nice job! Only ons small nitpick

File: video/video_decoder.cpp (hunk 1 above)
Comment (hari01584) [reply video/video_decoder.cpp:634]: done.

**Comment by sev-** [comment]
Thanks!

**Pull request merged**